getting_started/scripting/c_sharp/c_sharp_differences.rst

```diff
@@ -26,8 +26,8 @@ Math
 ----
 
 Math functions like ``abs``, ``acos``, ``asin``, ``atan`` and ``atan2`` are
-located under ``Mathf`` instead of in global scope.
-``PI`` is ``Mathf.PI``
+located under ``Mathf`` as ``Abs``, ``Acos``, ``Asin``, ``Atan`` and ``Atan2``, instead of in global scope.
+``Pi`` is ``Mathf.Pi``
```

Comment:
If the format is "<gdscript> is <csharp>" then should this be "PI is Mathf.Pi"?

Reply:
Yes, you are correct. I'll push a correction.